src/canvas/.gitignore

```diff
@@ -1,2 +1,3 @@
 # VS Code
 .vscode/
+*.csv
```

Comment:
Are you sure we need this file? Root level `.gitignore` already has `*.csv`, and generally we _don't_ want to ignore `.vscode` directories.

Reply:
@stephenfuqua  On un-ignore of .vscode, can only do if "python.pythonPath" is relative path.  pipenv is absolute path in user directory by default.  Poetry setting puts in a .venv folder in workspace, making path ".venv\\Scripts\\python.exe" work for everyone.

Reply:
`"python.venvPath": "${workspaceFolder}/.venv/bin/python"` is what we have in the root `.vscode` folder.

Reply:
After giving Poetry the "in project" config, it created a directory scheme with python.exe in .venv\Scripts.  VS Code picked saw it and added `"python.pythonPath":` ".venv\\Scripts\\python.exe",`